tests/test_gpu_examples.py

```diff
@@ -4172,18 +4173,19 @@ def test_hotswapping_compiled_model_does_not_trigger_recompilation(self):
         assert "__recompiles" in stderr.decode()
 
     @pytest.mark.xfail(strict=True, reason="Requires hotswap to be implemented in diffusers")
-    def test_hotswapping_compiled_diffusion_model_does_not_trigger_recompilation(self):
+    @pytest.mark.parametrize("ranks", ["7,13", "13,7"])  # the ranks of the 2 LoRAs as str
```

Comment:
Should we also check for different alphas?

Reply:
In `run_compiled_model_hotswap.py`, we set:

```py
    config0 = LoraConfig(init_lora_weights=False, r=rank0, lora_alpha=alpha0, target_modules=["q_proj", "v_proj"])
    config1 = LoraConfig(init_lora_weights=False, r=rank1, lora_alpha=alpha1, target_modules=["q_proj"])
```

so the alphas are indeed different.